mimesis/datasets/ar/address.json

```diff
@@ -0,0 +1,296 @@
+{
+  "address_fmt": "{st_num} {st_sfx} {st_name}",
+  
+  "city": [
+
+ 
+  "عين الأربعاء",
+  "عين الإبل",
+   "عين الباردة",
+  "عين البرد",
+  "عين البنيان",
+   "عين البيضاء",
+    "عين الترك",
+    "عين التركي",

```

Comment:
I know Arabic is a right-to-left (RTL) language, but the indentation seems incorrect here. Could you please format all these files using JSON Online Editor or a similar tool?

Reply:
ok, done